.devcontainer/init-services.sh

```
@@ -0,0 +1,26 @@
+        -p 5432:5432 \
+        quay.io/enterprisedb/postgresql \
+        -c listen_addresses='*'
+
+podman start sippy-redis 2>/dev/null || \
+    podman run -d --name sippy-redis \
+        --network sippy-net \
+        -p 6379:6379 \
+        redis:7-alpine
+
+echo "Waiting for PostgreSQL..."
+for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)